src/viur/shop/skeletons/article.py

```diff
@@ -85,6 +87,15 @@ def shop_is_low_price(self) -> BooleanBone:
         """shop_price_retail != shop_price_recommended"""
         ...
 
+    shop_view_url = RawBone(
```

Comment:
Why not use the [UriBone](https://github.com/viur-framework/viur-core/blob/main/src/viur/core/bones/uri.py) ? This means you can easily parameterise it in future.

Reply:
Hmm. I hadn't even thought about that. But the data is provided by the backend anyway, so all the validation is actually superfluous. Especially if the URL is only absolute without a host.